### PR TITLE
Firststeps orientation

### PR DESCRIPTION
**Pull request by carolanitz**
### Checklist
- [x] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
- [x] I've followed the [vlc-ios code style](Docs/CodingStyle.md)
- [x] I've read the [Contribution Guidelines](https://github.com/videolan/vlc-ios#contribute)
- [x] I've updated the documentation if necessary.

### Description

You will not be able to have the icons fully visible on smaller devices  but we can do follow ups where we remove the top bar and adjust the text to mitigate this.

Simulator Screen Shot - iPhone Xs - 2019-07-10 at 13 39 13

### CODE REVIEW

File: vlc-ios/AppUtitlity.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
+//
+//  AppUtitlity.swift
+//  VLC-iOS
+//
+//  Created by Carola Nitz on 10.07.19.
+//  Copyright © 2019 VideoLAN. All rights reserved.
+//
```
Comment (Mikanbu) [inline vlc-ios/AppUtitlity.swift]: I believe that the header style should be the one we generally use in this project.

Comment (carolanitz) [reply vlc-ios/AppUtitlity.swift]: oh yeah Thanks! I'll update

**Comment by carolanitz** [comment]
> LGTM, small nits inline.
> 
> The commit message should be changed: `Added` -> `Add`
> 
> Additionally, I believe that `AppUtitlity.swift` should be inside `Sources/`

done and done :) merging

**Comment by carolanitz** [comment]
merged with 01f6338a9c9cf5a1fb21314277228498965a0064 & 46156a42a598a77d70f079802dde359bda50de5a

**Pull request closed without merging**